config/application.rb

```diff
@@ -61,5 +63,8 @@ class Application < Rails::Application
 
     # Version of your assets, change this if you want to expire all your assets
     config.assets.version = "1.0"
+
+    # Force requests from old versions of IE to be UTF-8 encoded.
+    config.action_view.default_enforce_utf8 = true
```

Comment:
Do we still support IE <= 8?

Reply:
Yes, we still support IE 8.

Reply:
It turns out we can avoid this config anyway.